### PR TITLE
[EC-267] Remove unassigned from lists it is not suppsed to be in

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

The `Unassigned` collection was mistakenly added to lists other than the filters menu. This removes it.

## Code changes

All changes are just to filter collections with `null` `id`

## Before you submit

```
- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

File: libs/angular/src/components/add-edit.component.ts [hunk 1]
```diff
@@ -551,7 +551,7 @@ export class AddEditComponent implements OnInit {
 
   protected async loadCollections() {
     const allCollections = await this.collectionService.getAllDecrypted();
-    return allCollections.filter((c) => !c.readOnly);
+    return allCollections.filter((c) => !c.readOnly && c.id != null);
```
Comment (eliykat) [inline libs/angular/src/components/add-edit.component.ts:554]: This is coming from local synced data, which will never have the `Unassigned` collection (given that we only create it in `VaultFilterService.buildAdminCollections` for the purpose of showing it as a filter). So this check shouldn't be necessary here. (If it is then we have bigger problems)

File: libs/angular/src/components/collections.component.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ export class CollectionsComponent implements OnInit {
   protected async loadCollections() {
     const allCollections = await this.collectionService.getAllDecrypted();
     return allCollections.filter(
-      (c) => !c.readOnly && c.organizationId === this.cipher.organizationId
+      (c) => !c.readOnly && c.organizationId === this.cipher.organizationId && c.id != null
```
Comment (eliykat) [inline libs/angular/src/components/collections.component.ts:85]: Same here

**Review by eliykat — CHANGES_REQUESTED** [review]

File: libs/angular/src/components/add-edit.component.ts (hunk 1 above)
Comment (coroiu) [reply libs/angular/src/components/add-edit.component.ts]: Removed!

File: libs/angular/src/components/collections.component.ts (hunk 2 above)
Comment (coroiu) [reply libs/angular/src/components/collections.component.ts]: Removed!

File: apps/desktop/src/app/accounts/settings.component.html [hunk 3]
```diff
@@ -109,6 +109,10 @@ <h2>
                   </label>
                 </div>
               </div>
+
+              <div class="form-group">
+                <label>{{ "deleteAccount" | i18n }}</label>
+              </div>
```
Comment (eliykat) [inline apps/desktop/src/app/accounts/settings.component.html:115]: I think this is an accidental inclusion, please undo.

Comment (eliykat) [inline apps/desktop/src/app/accounts/settings.component.html:115]: I think this is an accidental inclusion, please undo.

**Review by eliykat — CHANGES_REQUESTED** [review]

**Review by eliykat — APPROVED** [review]

**Pull request merged**